### ISSUE TITLE
Preferences Dialog: Duplicate Shortcut Keys for Options

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#8913]
When taking a look at the options listed throughout the Preferences dialog, I've noticed that there are duplicate shortcut keys tied to individual options. I can list these if required - do we need to rethink keys that can be used for these?

**Comment by ehollig** [comment]
@elliott94 we may need to rethink these shortcuts. Could you post a list of shortcuts that may be duplicated as well as a suggested shortcut key for the option?

**Comment by elliott94** [comment]
After taking a look, I'm wondering if we might need to rethink which keys we have shortcuts bound to. Is the general idea that as many controls have shortcuts tied to them as possible, or do we just tie shortcuts to ones that make sense? As an example, in the General section of Settings there are several checkboxes that don't have shortcuts tied to them at all.

**Comment by Adriani90** [comment]
in my view at least I would say it does not make sense to add a shortcut for every setting, this will not be possible without duplicates or letters that do not match at all the words.
cc: @XLTechie, @CyrilleB79

**Comment by XLTechie** [comment]
I also wonder what the logic is, behind having shortcuts for some of them, but 
not for others. Is this just accident, or does it serve some design goal of 
which I am unaware?

It would be difficult to find unique shortcuts that appear in the words, for 
each possible option. Especially for the larger lists of options, such as 
Document settings.

Still, for new shortcuts, or for old options that are getting new shortcuts, I 
agree that every effort should be made to find unique shortcuts when possible.

But I don't know if we should go around changing existing shortcuts that much. 
As I recently pointed out in #15364, it has the potential to break a lot of 
long-written support instructions.

**Comment by Adriani90** [comment]
It will most probably not be possible to avoid all duplicate shortcuts. There are much more words than letters, so no way to add an unique letter to really every setting.
I am closing as can't fix. if you are having a specific problem with a setting that you think should change its shortcut, please feel free to open a new request.

**Comment by CyrilleB79** [comment]
A unique shortcut may not be possible in all panels, but progress can probably be made to avoid duplicates as much as possible and/or to add more shortcuts to options which haven't.

But before asking such modifications, it would be nice to have a consistent definition of the desired goal:
* should all options have a shortcut?
* if not, what should be the criterion to have a shortcut or not?
* should duplicates be avoided?
* if not possible, how such conflicts should be handled?
* etc.

To summarize, define guidelines for accelerator keys could be interesting. Cc @Qchristensen or @gerald-hartig in case you are interested by the topic.

**Comment by gerald-hartig** [comment]
@CyrilleB79 Happy to take this further, but at first glance this looks like a low impact issue and drafting the guidelines and getting community alignment would be a bit of work.
